spec/minimax_spec.rb

```diff
@@ -0,0 +1,85 @@
+require_relative '../lib/minimax.rb'
+require_relative '../lib/three_by_three_ttt_board.rb'  
+require_relative '../lib/three_by_three_ttt_rules.rb'
+
+describe Minimax do 
```

Comment:
I'm finding it hard to tell what each test is covering. Can you shorten/clarify the test descriptions? It's safe to remove the return value information because that's part of your expectation.